docs/getting-started.md

```diff
@@ -68,33 +68,30 @@ You can copy and paste this example:
 
 #### Use the Bit component in your code
 
-3\. Install the bit-js module using [NPM](https://www.npmjs.com/package/bit-js) or Yarn.
+Bit is designed to be language agnostic, so in order to use components, you first need to install a [language-specific driver](drivers.md) for Bit. This is to enable each specific programming language to be able to use components.
```

Comment:
> Bit is designed to be language agnostic, so in order to use components, you first need to install a [language-specific driver](drivers.md) for Bit. This is to enable each specific programming language to be able to use components.

Bit designed to be language agnostic, so in order to use the components, you first need to install a [language-specific driver](drivers.md).